lib/tasks/evm.rake

```diff
@@ -56,6 +56,12 @@ namespace :evm do
     puts inventory.tableize if inventory.present?
   end
 
+  desc "Determine if the configured encryption key is valid"
+  task :validate_encryption_key => :environment do
```

Comment:
Would loading environment as a prerequisite possibly blow up due to a bad encryption key before we get to this task?

Reply:
I can run `bundle exec rake environment` with the wrong key, so I think this is fine.